wobgob/azerothcore-wotlk
# Loot Boxes on Milestone Levels

## Changes Proposed:
-  This module will give the player a Level n Loot Box at levels: 10, 19, 29, 39, 49, 59, 60, 64. (Hereafter referred to as milestone levels)

## Issues Addressed:

- Closes #27 

## Tests Performed:

- Tested on WoW 3.3.5a using Windows 10 Pro x64
- All milestone levels grant a loot box via ingame mail.


## How to Test the Changes:


1. Level a character to a milestone level.
2. A system message will appear prompting you to check your mail for a reward
3. Grab the level loot box from your mail, and open it.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
